server.go

```diff
@@ -2332,6 +2333,24 @@ func (s *server) Stop() error {
 		if err := s.htlcNotifier.Stop(); err != nil {
 			srvrLog.Warnf("failed to stop htlcNotifier: %v", err)
 		}
+
+		// Update channel.backup file. Make sure to do it before
```

Comment:
The primary goal of updating `channel.backup` before shutdown is to update the commitment transaction to the most recent version. Because of this, if LND was shutdown properly, the versions of the close transactions in the backup are always the latest ones and it is safe to use that channel backup to force close a channel if LND hasn't started after that. This is a convenient property for those who runs LND only when needed. They can backup `channel.backup` after each LND using (and shurring down) and be self-soverein in their funds.

Reply:
TBH, I think it also makes sense to add a timestamp here making it more tangible for a user whether this is an outdated closeTx